### PR TITLE
Asintel 2028 comt sv implementation

### PR DESCRIPTION
**Pull request by sarmaevanidfe**

### CODE REVIEW

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql [hunk 1]
```diff
@@ -128,10 +128,14 @@ VALUES
...
 ,('Commitments','Providers','dbo','[Created],[Updated]','','[Ukprn],[Name]','Comt_Providers',1)
 ,('Commitments','DataLockStatus','dbo','[Id],[DataLockEventId],[DataLockEventDatetime],[PriceEpisodeIdentifier],[ApprenticeshipId],[IlrTrainingCourseCode],[IlrTrainingType],[IlrActualStartDate],[IlrEffectiveFromDate],[IlrPriceEffectiveToDate],[IlrTotalCost],[ErrorCode],[Status],[TriageStatus],[ApprenticeshipUpdateId],[IsResolved],[EventStatus],[IsExpired],[Expired]','','','Comt_DataLockStatus',0)
 ,('Commitments','ApprenticeshipConfirmationStatus','dbo','[ApprenticeshipId],[ApprenticeshipConfirmedOn],[CommitmentsApprovedOn],[ConfirmationOverdueOn]','','','Comt_ApprenticeshipConfirmationStatus',0)
+,('Commitments','Standard','dbo','[StandardUId],[LarsCode],[IFateReferenceNumber],[Version],[Level],[Duration],[MaxFunding],[EffectiveFrom],[EffectiveTo],[VersionMajor],[VersionMinor],[StandardPageUrl],[Status],[IsLatestVersion]','','[Title]','Comt_Standard',1)
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql:136]: Has it not been possible to model standards Sarma?

**Review by uhimabindu — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql (hunk 1 above)
Comment (sarmaevanidfe) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateSourceDbMetadataForImport.sql:136]: Now comt Standard is modelled and merged with base [FAT2_StandardSector]

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportFAT2SectorStandardToPL.sql [hunk 2]
```diff
@@ -79,7 +79,12 @@ BEGIN TRY
 								,[TypicalDuration]
 								,[MaxFunding]
 								,[IsActive]
-								,[VersionApprovedForDelivery]
+								,[VersionApprovedForDelivery]	
+								,[EffectiveFrom]
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportFAT2SectorStandardToPL.sql:83]: I thought we discussed we will remove EffectiveFrom and EffectiveTo Fields?

File: src/SFA.DAS.DATA.MANAGEMENT/Tables/ASDataPL_FAT2_SectorStandard.sql [hunk 3]
```diff
@@ -39,5 +39,10 @@
 	[MaxFunding]									[Int]				NULL,	
 	[IsActive]										[Bit]				NULL,
 	[VersionApprovedForDelivery]					[DateTime]			NULL, 
+	[EffectiveFrom]									[DateTime]			NULL, 
+	[EffectiveTo]									[DateTime]			NULL,
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/Tables/ASDataPL_FAT2_SectorStandard.sql:43]: same here? Thought we wanted to remove these two fields?

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateMetadataRolesAndPermissions.sql [hunk 4]
```diff
@@ -85,6 +85,9 @@ VALUES
 ('BetaUser','ASData_PL','Comt_Apprenticeship','TABLE','SELECT',0,1),
 ('BetaUser','ASData_PL','Comt_Commitment','TABLE','SELECT',0,1),
 ('BetaUser','ASData_PL','Comt_ApprenticeshipConfirmationStatus','TABLE','SELECT',0,1),
+('BetaUser','ASData_PL','Comt_ApprenticeshipUpdate','TABLE','SELECT',0,1),
+('BetaUser','ASData_PL','Comt_StandardFunding','TABLE','SELECT',0,1),
```
Comment (uhimabindu) [inline src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/PopulateMetadataRolesAndPermissions.sql:89]: No Standard Funding Table right?

**Review by uhimabindu — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportFAT2SectorStandardToPL.sql (hunk 2 above)
Comment (sarmaevanidfe) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportFAT2SectorStandardToPL.sql:83]: EffectiveFrom and EffectiveEndDates are different VersionLatestStart and EndDate , so EffectiveFrom and EffectiveEndDates are added back to schema

**Comment by karthik-dfe** [comment]
Old PR

**Pull request closed without merging**